en/hosting_guide/requirements.rst

```diff
@@ -1,14 +1,26 @@
 Requirements
 ============
 
-Webserver
----------
+Front servers
+-------------
+
+Production server
+~~~~~~~~~~~~~~~~~
 
 It's required to have a PHP webserver in order to run Open Orchestra.
 
 * PHP 5.4+
 * Apache 2 or NginX
 * Varnish 3 reverse proxy (optionnal)
+* Nodejs
```

Comment:
Ok then I think the menu is not optimal : what interest a sys admin requested to prepare an open orchestra environnement is to get a short and exhaustive list of what is required and a second one with what is optional. Two short bullet list.

The menu you're proposing is more a functional one and corresponds to an explanation of who does what. That does not matter to a sys admin. He only want to know what tool on which version.